java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestItemUtils.java

```diff
@@ -52,7 +52,8 @@ public static String parseFullName(IJavaElement element, TestLevel level, TestKi
                     // Generics don't come through in the test results, so we need to strip
                     // them out now.
                     final String methodName = JavaElementLabels.getElementLabel(element, JavaElementLabels.ALL_DEFAULT)
-                            .replaceAll("<.*?>", "");
+                            .replaceAll("<.*?>", "")
+                            .replaceAll(" ", "");
```

Comment:
is tab legal? how about using \\s ?

Reply:
Take a rough look into the upstream implementation, only ` ` is used. Should be safe here only matching spaces